### PR TITLE
[Docker] Making sure cache is empty in distribution image

### PR DESCRIPTION
**Pull request by vidarl**

### CODE REVIEW

File: doc/docker/Dockerfile-distribution [hunk 1]
```diff
@@ -5,6 +5,8 @@ FROM docker_app as builder
 RUN composer config extra.symfony-assets-install hard
 RUN composer run-script post-install-cmd --no-interaction
 
+RUN rm -Rf /var/www/var/cache/prod/*
```
Comment (alongosz) [inline doc/docker/Dockerfile-distribution]: @vidarl what if someone setups dev env instead of prod? Or is always prod at this level?

Comment (vidarl) [reply doc/docker/Dockerfile-distribution]: well, you actually may change to dev.
I have updated PR to delete any dev and test cache too

Comment (alongosz) [reply doc/docker/Dockerfile-distribution]: Hmm, then I think I have a simpler suggestion:
`RUN rm -Rf /var/www/var/cache/*/*`
WDYT?

Comment (vidarl) [reply doc/docker/Dockerfile-distribution]: yeah. that would do the trick too

Fixed

**Pull request merged**